examples/hitl/rearrange_v2/rearrange_v2.py

```diff
@@ -316,6 +328,8 @@ def _update_help_text(self):
             self._app_service.text_drawer.add_text(
                 status_str,
                 TextOnScreenAlignment.TOP_CENTER,
+                text_delta_x=-280,
```

Comment:
lets make these configurable. I imagine the number is different for different displays?

Reply:
I wouldn't stress about it too much right now.

This `text_drawer` class has pretty rudimentary text layout formatting. For instance, we cannot do things like text alignment or precise positioning.

We're looking to improve formatting here to avoid this kind of fiddling.